sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample13_RecognizeInvoicesFromFile.cs

```diff
@@ -34,36 +34,6 @@ public async Task RecognizeInvoicesFromFile()
 
                 foreach (RecognizedForm invoice in invoices)
                 {
-                    FormField invoiceIdField;
-                    if (invoice.Fields.TryGetValue("InvoiceId", out invoiceIdField))
```

Comment:
Removing some fields just to make it smaller so the Readme doesn't look that big...